data/korath/korath culture conversations.txt

```diff
@@ -0,0 +1,76 @@
+
+mission "Korath Far'en Lai Prayer"
+	minor
+	invisible
+	to offer
+		day >= 13
+		day < 18
+		random < 35
+	source
+		attributes efret
+		not attributes station
+	on offer
+		conversation
+			`The common gloomy feel of the local spaceport appears to be somehow even more pronounced today, with very few Korath at all doing anything but what is absolutely necessary for the local facilities, as if they all had their attention elsewhere. It doesn't take you long to notice the gathering of a considerable number of Korath by one large, blackened side of an abandoned building, which seems to be the reason for the drop in the local morale.`
+			`	As you approach the group, you notice a few Korath children running their hands over the dark wall, scratching it carefully with what must be something akin to crayons, or some material that could perform the same function. After they play around for a while, you notice all their drawings seem to take a similar shape: a beige or brownish circle, with some shy spots of blue, and rare bits of green. Contrasted by the onyx background of the building wall, you easily make it out to be a planet.`
+			`	Up close, you see many adult and elderly Korath, gathered as if in a prayer group around a shrine of sorts, just a few steps away from the building. They are mostly silent, but for a few brief gasps. The shrine seems to be simply a pile of common crates and boxes, hidden by a dusty canvas that's not large enough to cover all of the cubic components of the pyramid-like structure.`
+			`	At the bottom of the shrine, the Korath seem to have deposited a plethora of personal belongings, from clothing pieces to tools, pictures and coins, books and crystals, large fragments of cracked eggshells, flasks of oils or other substances, small jewelry pieces, and all manner of beads. The items seem to pile on and on endlessly, as every passing Korath in the spaceport makes a point to pass by the shrine, and leave something as an offering.`
+			`	On the first level of crates, you first see a number of wooden, stone and metal slabs, with various scratch marks over them. Upon closer inspection, however, you see that from certain angles, the marks combine to form shapes, like those of the planet drawn by the children, but also of many other planets, with clearly defined oceans and continents. There are stars, systems, symbols you can't recognize, and a great number of tablets with nothing but writing.`
+			`	One of the oldest Korath in the gathering, who had been shivering with his head in his hands since before you approached, lets out an agonizing, bark-like yelp, in a frayed voice, as he looks up to the top of the shrine. Standing at navel height, one lone crate supports only a few items. A large, colorful feather, an amazingly complex seashell, and a mysteriously beautiful flower, sit at the top crate, all surrounding one single, lone candle, which looks as if it's been burning for hours on end.`
```

Comment:
```suggestion
			`	One of the oldest Korath in the gathering, who had been shivering with his head in his hands since before you approached, lets out an agonizing, bark-like yelp, in a frayed voice, as he looks up to the top of the shrine. Standing at navel height, one lone crate supports only a few items. A large, colorful feather, an amazingly complex seashell, and a mysteriously beautiful flower sit on the top crate, all surrounding one single, lone candle, which looks as if it's been burning for hours on end.`
```